docs/backup-restore.md

```diff
@@ -127,6 +133,12 @@ You must enable the PITR feature for DynamoDB tables. If you're using [ScalarDB
 
 To specify a transactionally consistent restore point, pause your application that is using ScalarDB with DynamoDB as described in [Back up with explicit pausing](#back-up-with-explicit-pausing).
 </div>
+<div id="YugabyteDB_Managed2" class="tabcontent" markdown="1">
+
+Clusters are backed up automatically based on the backup policy, and these backups are retained for a specific duration. You can also perform on-demand backups. See [YugabyteDB Managed: Back up and restore clusters](https://docs.yugabyte.com/preview/yugabyte-cloud/cloud-clusters/backup-clusters/).
+
+If you're using YugabyteDB with other databases, to ensure transactional consistency across databases, you'll need to pause the application using ScalarDB or [ScalarDB Cluster (redirects to the Enterprise docs site)](https://scalardb.scalar-labs.com/docs/latest/scalardb-cluster/) during the backup process, as described in [Back up with explicit pausing](#back-up-with-explicit-pausing).
```

Comment:
Just a minor revision to reduce eliminate. (I originally read this part as `pause the application by using ScalarDB`, which I believe is incorrect.)

```suggestion
If you're using YugabyteDB with other databases, to ensure transactional consistency across databases, you'll need to pause the application that is using ScalarDB or [ScalarDB Cluster (redirects to the Enterprise docs site)](https://scalardb.scalar-labs.com/docs/latest/scalardb-cluster/) during the backup process, as described in [Back up with explicit pausing](#back-up-with-explicit-pausing).
```